conan/cli/commands/graph.py

```diff
@@ -192,6 +192,9 @@ def graph_info(conan_api, parser, subparser, *args):
                            help="Deployer output folder, base build folder by default if not set")
     subparser.add_argument("--build-require", action='store_true', default=False,
                            help='Whether the provided reference is a build-require')
+    subparser.add_argument("--out-file", action=OnceArgument,
```

Comment:
This should be injected automatically and conditionally to the existence of formatters together with the definition of the ``--format`` argument. There cannot be a ``--out-file`` without a ``--format`` definition.

Reply:
Finally, my proposal is to add the `--out-file` always, to enable users do for example
``
conan config home --out-file=file.ext 
``
and it uses the text formatter that is the one by default. Because in the end, there's always a formatter that is the `text` one even if it does not show in the `--help`, but let's discuss about this with the team.